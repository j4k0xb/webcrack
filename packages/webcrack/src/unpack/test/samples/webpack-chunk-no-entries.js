(window['webpackJsonp'] = window['webpackJsonp'] || []).push([
  [1],
  [function (module, exports, __webpack_require__) {}],
  [[0, 0]],
]);
