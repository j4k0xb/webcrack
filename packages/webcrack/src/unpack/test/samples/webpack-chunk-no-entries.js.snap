WebpackBundle {
  "entryId": "",
  "modules": Map {
    "0" => WebpackModule {
      "ast": ,
      "id": "0",
      "isEntry": false,
      "path": "./0.js",
    },
  },
  "type": "webpack",
}